mirror-godot-app/ui/common/components/endless_scroll_flow_container.gd

```diff
@@ -69,6 +69,11 @@ func _calculate_max_items_on_screen(forced_size = Vector2.ZERO) -> int:
 
 
 func _fetch_data(cnt_items_to_fetch: int) -> Array:
+	if not fetch_callable.is_valid():
+		# this could be caused by the data not being initialised before this is called
+		# it happens generally if you're not logged in before populating this.
+		push_error("invalid fetch callable in endless scroll container")
```

Comment:
I found another bug with this returning one element instead of the correct quantity i will investigate further